src/controllers/usersController.js

```diff
@@ -0,0 +1,12 @@
+const User = require("../models/User");
+
+// @desc    Endpoint for fetching all the registered users
+// @route   GET /api/v1/users
+// @access  public
+const getUsers = async (req, res) => {
+  const registeredUsers = await User.find({}, "name email");
+
+  res.json({ count: registeredUsers.length, users: registeredUsers });
+};
+
+module.exports = { getUsers };
```

Comment:
Under normal circumstances, I think it would be best to put this in a separate ticket and PR since it's kind of unrelated functionality, but I suspect you added this because the frontend expects it to be there right now, and since we're in "ship now, ask questions later" mode... 👍😄 

(Also, it's not my favorite thing to allow any logged in user to see a list of all other users that exist in the app since it seems like a violation of user privacy to me. But again, probably not the time to nitpick about this. 🙂 )

Reply:
Yep exactly my thought as well. Only planning on keeping this endpoint until email based invitation is added.